api/osrd_infra/schemas/study.py

```diff
@@ -0,0 +1,27 @@
+from enum import Enum
+
+
+class StudyState(str, Enum):
+    """
+    This class allows to know the state of the operational study.
+    """
+
+    started = "started"
```

Comment:
It is recommended by the official doc to use snake cases, upper cases names for enum members https://docs.python.org/3/howto/enum.html

Reply:
Nicolas wanted me to return it this way.

Reply:
```suggestion
    STARTED = "started"
```
and if you do `StudyState.STARTED.value` you get the correct `"started"` no ?